File: core-java-practice/gcr-codebase/PerimeterOfRectangle.java
import java.util.Scanner;

public class PerimeterOfRectangle {
    public static void main(String[] args) {
        //read input
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter the length: ");
        int length=sc.nextInt(); //take length as input

        System.out.println("Enter the breadth: ");
        int breadth=sc.nextInt(); //take breadth as input

        //calculate the perimeter
        int perimeter=2*length*breadth;

        System.out.print("The perimeter of the rectangle is: "+ perimeter);

    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/PerimeterOfRectangle.cs
using System;

class PerimeterOfRectangle{
    public static void Main(string[] args){

        Console.WriteLine("Enter the length of the rectangle: ");
        int length=int.Parse(Console.ReadLine()); //taking length as an input

        Console.WriteLine("Enter the breadth of the rectangle: ");
        int breadth=int.Parse(Console.ReadLine());

        //calculating the perimeter
        int perimeter=2*(length+breadth);

        //printing the perimeter
        Console.Write("The perimeter of the rectangle is: "+ perimeter);

    }
}